house/models.py

```diff
@@ -49,8 +50,15 @@ class House(models.Model):
     house_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     name = models.CharField(max_length=50)
     public = models.BooleanField(default=False)
-    country = models.ForeignKey(Country, on_delete=models.SET_NULL, null=True)
-    city = models.ForeignKey(City, on_delete=models.SET_NULL, null=True)
+    country = models.ForeignKey(Country, on_delete=models.CASCADE, null=True)
+    city = ChainedForeignKey(
```

Comment:
Nice Solution to the problem
i think that it is better then what I tried to do
I tried to fix the country-city problem by Implementing a Dependent/Chained Dropdown List with Django
and adding it to the django form creating a house, but i haven't managed to make it to work...
Good Job Lior 👍

Reply:
Thanks Lior, this was a bit of work mainly because of no support for our version of Django and python and the way we initiated the countries and cities data. so this comment is much appreciated 🥇